src/reset/readme.md

```diff
@@ -20,15 +20,27 @@ The method allow to reset many stores by a single line
 
 ```ts
 reset({ clock, target });
+
+or
+
+const resetEvent = reset({ target });
```

Comment:
I propose to split definitions on two sections.
You can review the example here: https://patronum.effector.dev/methods/every/

For each override we have separate section.
<img width="319" alt="image" src="https://user-images.githubusercontent.com/5620073/218976433-897620df-0166-428a-be52-fd6502d0f288.png">

Reply:
Done 😏